src/test/java/org/apache/datasketches/memory/internal/BufferTest.java

```diff
@@ -60,56 +59,56 @@ public void checkDirectRoundTrip() throws Exception {
   public void checkAutoHeapRoundTrip() {
     int n = 1024; //longs
     WritableBuffer wbuf = WritableMemory.allocate(n * 8).asWritableBuffer();
-    for (int i = 0; i < n; i++) {
-      wbuf.putLong(i);
-    }
-    wbuf.resetPosition();
-    for (int i = 0; i < n; i++) {
-      long v = wbuf.getLong();
-      assertEquals(v, i);
+      for (int i = 0; i < n; i++) {
+        wbuf.putLong(i);
+      }
+      wbuf.resetPosition();
+      for (int i = 0; i < n; i++) {
+        long v = wbuf.getLong();
+        assertEquals(v, i);
+      }
     }
-  }
 
   @Test
   public void checkArrayWrap() {
     int n = 1024; //longs
     byte[] arr = new byte[n * 8];
     WritableBuffer wbuf = WritableMemory.writableWrap(arr).asWritableBuffer();
-    for (int i = 0; i < n; i++) {
-      wbuf.putLong(i);
-    }
-    wbuf.resetPosition();
-    for (int i = 0; i < n; i++) {
-      long v = wbuf.getLong();
-      assertEquals(v, i);
-    }
+      for (int i = 0; i < n; i++) {
```

Comment:
what happened to alignment here and below? tabs used?

Reply:
Wow! this is weird. No tabs used.  Not sure how this happened.  We have had two folks editing these files recently, perhaps they used a different indention scheme.